es6-shim.js

```diff
@@ -1498,7 +1498,11 @@
     },
 
     log2: function (value) {
-      return Math.log(value) * Math.LOG2E;
+      if (value === 0) {
+        return -1 / 0;
+      }
+      var e = Math.min(Math.floor(Math.log(value) / Math.LN2), 1023);
```

Comment:
instead of 1023 there should be `Math.floor(Math.log(Number.MAX_VALUE) / Math.log(2))` constant